app/controllers/computacenter/school_changes_controller.rb

```diff
@@ -0,0 +1,89 @@
+class Computacenter::SchoolChangesController < Computacenter::BaseController
```

Comment:
It'd be nice to add Pundit authorisation to this controller. It's not strictly necessary because `Computacenter::BaseController` already does authorisation, but adding it now makes the job of moving to Pundit eventually easier.

Reply:
Done in ce7e664f